spec/cassia/access_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
+
+RSpec.describe Cassia::AccessController do
+  include FaradayHelpers
+  
+  describe "#get_token" do
+    vcr_options = { cassette_name: 'access_controller/get_token/success', record: :new_episodes }
```

Comment:
below you are building the `response_body`, if that is true you shouldn't need any cassettes. I think we should go with the cassettes though instead of building the response our selves